internal/cli/terraform.go

```diff
@@ -187,3 +216,51 @@ import {
 
 	return t.Execute(file, data)
 }
+
+func generateTerraformResourceConfig(ctx context.Context, outputDIR string) error {
```

Comment:
Quite hard to unit test this, as the logic here downloads a terraform binary, and then expects to be able to run terraform init and plan by having real auth0 credentials available. 

We can deferr to E2E tests instead for this in a following PR.